docs/docs-content/vm-management/create-manage-vm/advanced-topics/deploy-import-ova.md

```diff
@@ -254,18 +255,18 @@ name, for example `cdi-uploadproxy.mycompany.io`, to the Nginx load balancer’s
-    | **CDI Upload Proxy URL**                    | Optionally provide a custom CDI upload proxy URL. If ingress is configured for the CDI upload proxy, the ingress hostname will be used by default and must be resolvable via DNS. If the CDI upload proxy is exposed via a NodePort, a node IP and ephemeral port will be used. Depending on how CDI and DNS are configured, you may need to edit `/etc/hosts` to ensure DNS resolution. You may also port-forward the CDI upload proxy via `kubectl -n cdi port-forward deployment/cdi-uploadproxy 8443` and provide `https://localhost:8443` as the CDI upload proxy URL; however this approach will be less efficient. |                                   |
+    | **Parameter**                               | **Description**                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     | **Values**                        |
+    | ------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------- |
+    | **OVA Path**                                | The path to the image your have uploaded to your VM. The path for the example provided is `/root/bitnami-wordpress-6.2.2-r1-debian-11-amd64.ova/`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |                                   |
```

Comment:
Suggestion: "The path to the image your have uploaded" -> "The path to the OVA you have uploaded..."